src/Microsoft.AspNetCore.DataProtection.EntityFrameworkCore/DataProtectionKey.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Microsoft.AspNetCore.DataProtection.EntityFrameworkCore
+{
+    /// <summary>
+    /// Code first model used by <see cref="EntityFrameworkCoreXmlRepository{TContext}"/>.
+    /// </summary>
+    public class DataProtectionKey
+    {
+        /// <summary>
+        /// The entity identifier of the <see cref="DataProtectionKey"/>.
+        /// </summary>
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
```

Comment:
`int Id` property will be mapped as "identity" key by convention, so these attributes are not needed by EF.